### PR TITLE
has_raced attribute added to show serializer

### PR DESCRIPTION
**Pull request by larathordar**
#### Changes proposed in this pull request:

* Added  has_raced attribute to show serializer
#### What I have learned working on this feature:
* To add an attributte to a serializer

#### Screenshots:

screen shot 2018-04-08 at 14 49 00

### CODE REVIEW

**Review by jyuw — APPROVED** [review]

File: db/schema.rb [hunk 1]
```diff
@@ -36,36 +36,6 @@
...
-    t.string "email"
-    t.text "tokens"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["confirmation_token"], name: "index_admins_on_confirmation_token", unique: true
-    t.index ["email"], name: "index_admins_on_email", unique: true
-    t.index ["reset_password_token"], name: "index_admins_on_reset_password_token", unique: true
-    t.index ["uid", "provider"], name: "index_admins_on_uid_and_provider", unique: true
-  end
-
```
Comment (holgertidemand) [inline db/schema.rb:68]: Why is this removed? :)
Forgot to migrate? ;)

Comment (larathordar) [reply db/schema.rb:68]: I did migrate ... that's weird

Comment (larathordar) [reply db/schema.rb:68]: I will try to migrate again

Comment (holgertidemand) [reply db/schema.rb:68]: Pull from upstream?

**Comment by holgertidemand** [comment]
looks good!

**Review by holgertidemand — APPROVED** [review]
